sdk/mixedreality/Azure.MixedReality.Authentication/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0001:Use one of the following pre-approved namespace groups (https://azure.github.io/azure-sdk/registered_namespaces.html): Azure.AI, Azure.Analytics, Azure.Communication, Azure.Data, Azure.DigitalTwins, Azure.Iot, Azure.Learn, Azure.Media, Azure.Management, Azure.Messaging, Azure.Search, Azure.Security, Azure.Storage, Azure.Template, Azure.Identity, Microsoft.Extensions.Azure", Justification = "<Pending>", Scope = "namespace", Target = "~N:Azure.MixedReality.Authentication")]
```

Comment:
@AlexGhiondea I forgot I had these suppressions. We'll be adding more clients to the `Azure.MixedReality` umbrella, so is it worth adding `Azure.MixedReality` to the list or would you prefer a different solution?

Reply:
We did have a discussion about this in the SDK API review and determined that the `Azure.MixedReality` name was necessary, so I don't expect we'll need to revisit that, but the tooling will need to be updated at some point.